hr_employee_service/__manifest__.py

```diff
@@ -2,26 +2,16 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 {
-    'name': 'HR Employee Service',
-    'version': '12.0.1.0.1',
-    'category': 'Human Resources',
-    'website': 'https://github.com/OCA/hr',
-    'author':
-        'Brainbean Apps, '
-        'Odoo Community Association (OCA)',
-    'license': 'AGPL-3',
-    'installable': True,
-    'application': False,
-    'summary': 'Employee service information & duration',
-    'depends': [
-        'hr',
-    ],
-    'external_dependencies': {
-        'python': [
-            'dateutil',
-        ],
-    },
-    'data': [
-        'views/hr_employee.xml',
-    ],
+    "name": "HR Employee Service",
+    "version": "13.0.1.0.0",
+    "category": "Human Resources",
+    "website": "https://github.com/OCA/hr",
+    "author": "Brainbean Apps, " "Odoo Community Association (OCA)",
```

Comment:
`" "` this looks not great?

@pedrobaeza can we do something about these cases with autoformatter?

Reply:
No, except manual change

Reply:
Do OCA has any opinion on that cosmetic issue?